platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/editor/StateEditorSave.java

```diff
@@ -78,7 +78,7 @@ public void run() {
                     DefaultConfiguration.buildBasicConfiguration(ConfigurationBuilder.create());
 
             final PlatformContext platformContext = new DefaultPlatformContext(
-                    configuration, new NoOpMetrics(), CryptographyHolder.get(), Time.getCurrent());
+                    configuration, new NoOpMetrics(), CryptographyHolder.get(), Time.getCurrent(), null);
```

Comment:
Should this be null? I think it would be ok to instantiate a file system manager in the current working directory for this.